commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/CommerceTableTag.java

```diff
@@ -215,15 +216,34 @@ protected int getMinPageSize() {
 	private void _setItems(String dataProviderKey) throws Exception {
 		Map<String, Object> context = getContext();
 
+		String tableName = GetterUtil.getString(context.get("tableName"));
+
+		if (tableName.equals(ParamUtil.getString(request, "tableName"))) {
```

Comment:
we should set this parameter `tableName` to a variable before checking for equality....

I guess just use `tableName1` and `tableName2` for the names